src/deepsparse/server/cli.py

```diff
@@ -165,11 +174,13 @@ def task(
     host: str,
     port: int,
     log_level: str,
+    loggers: str,
 ):
     """
     Run the server using configuration with CLI options,
     which can only serve a single model.
     """
+    loggers = loggers or None
```

Comment:
Recommend doing the loading of the yaml config in this function and passing the dictionary to server config.

```
loggers = {}
if loggers_path is not None:
    loggers = yaml.safe_load(...)
```

Reply:
was planning on empty dict meaning no loggers, `None` meaning use the default logger for the integration so not sure we'd want to unwrap like that here

Reply:
`None` being default loggers is really confusing from user perspective. recommend finding another way to specify default

Reply:
fair, the way to get the default would just be to omit `loggers` from the config, but agreed `null` vs empty dict isn't intuitive

Reply:
omitting loggers from config would set it as the default value right? which atm is None.

Thoughts on:
- loggers is omitted or set to None (null in yaml): no loggers `ManagerLogger(loggers=[])`
- loggers is a str or dict: current behavior of doing logger_manager_from_config
- ServerConfig.loggers Field.default is {"prometheus": {}}
- task cli loggers is an Optional[str] that defaults to None, meaning no loggers

Reply:
thinking through this proposal, need something that 1) gives a user an easy/default way to specify the default 2) let's us use the get default function (we'll need to change the default based on other info from the server config like the integration)

Reply:
updated, possible values:

`None` - no loggers
str - must be "default" for default (this is the default)
dict - config literal

additionally for simplicity in task CLI mode, logging is always default, added `no_logging` option to set loggers to None. (Otherwise would have required file support for a reasonable UX, can look into adding back in file support later as needed, but as discussed offline, probably not necessary for a while)

Reply:
Nice, i like it!